Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -130,18 +132,41 @@ extension Google_Protobuf_Duration: _CustomJSONCodable {
     }
 }
 
-
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
 
     public init(floatLiteral value: Double) {
+        // TODO: this rounds the nanos part towards zero, not to the nearest
```

Comment:
I actually changed this to match init(timeInterval:) but reverted that in favor of a discussion. Right now my position is that these two things should match, and that the implementation in init(timeInterval:) is correct. If you wind up doing some innocuous looking Double arithmetic and the answer is just epsilon less than 2, I think you want (2, 0) not (1, 999999999).